pkg/trace/config/config.go

```diff
@@ -287,10 +287,9 @@ type AgentConfig struct {
 	// Concentrator
 	BucketInterval         time.Duration // the size of our pre-aggregation per bucket
 	ExtraAggregators       []string      // DEPRECATED
-	PeerServiceAggregation bool          // TO BE DEPRECATED - enables/disables stats aggregation for peer.service, used by Concentrator and ClientStatsAggregator
 	PeerTagsAggregation    bool          // enables/disables stats aggregation for peer entity tags, used by Concentrator and ClientStatsAggregator
 	ComputeStatsBySpanKind bool          // enables/disables the computing of stats based on a span's `span.kind` field
-	PeerTags               []string      // additional tags to use for peer entity stats aggregation
+	PeerTags               []string      // additional tags to use for peer entity stats aggregation. If this field is not empty, the concentrator will aggregate on all of these keys.
```

Comment:
Is the comment still true? I think `PeerTagsAggregation` also needs to be enabled?